### PR TITLE
Add Plaid settings for Stripe gateway account

### PR DESCRIPTION
**Pull request by slavcodev**
image

### CODE REVIEW

**Review by adamaltman — DISMISSED** [review]

**Comment by vova07** [comment]
I'm pretty agree with this proposal, but I have some doubts even I don't have an alternative solutions.

My concern is related to the fact that `Plaid` MUST be used "de fatco" for all `ACH` transactions that goes through `Stripe` gateway.

In that way, from the proposal we get a case when we can select the `payment method = ACH` and save the gateway without any errors, but in fact we should send an alert and require the merchant to fill the `Plaid` credentials.

I propose to leave only `Plaid` credentials, and require them based on the selected payment method, without any needs in specifying the `Use Plaid` option at all.

What do you think?

**Comment by slavcodev** [comment]
I'm fine to remove `usePlaid` and add it when we will add other ways to process ACH payments

**Review by vova07 — DISMISSED** [review]

File: openapi/components/schemas/GatewayAccountConfig/Stripe.yaml [hunk 1]
```diff
@@ -12,6 +12,23 @@ allOf:
...
+                type: string
+                description: Plaid public key.
+              secret:
+                type: string
+                description: Plaid secret key.
+                format: password
+            required:
+              - clientId
+              - publicKey
+              - secret
```
Comment (slavcodev) [inline openapi/components/schemas/GatewayAccountConfig/Stripe.yaml:31]:
```suggestion
          clientId:
            type: string
            description: Plaid client ID is required for the `bank-account` payment method.
          publicKey:
            type: string
            description: Plaid public key is required for the `bank-account` payment method.
          secret:
             type: string
             description: Plaid secret key is required for the `bank-account` payment method.
             format: password
```

Comment (slavcodev) [reply openapi/components/schemas/GatewayAccountConfig/Stripe.yaml:31]: I think backend can't work with nested credentials keys.

Comment (vova07) [reply openapi/components/schemas/GatewayAccountConfig/Stripe.yaml:31]: At least it require additional adjustments in multiple places to handle that.

So I agree with new proposal.

Comment (slavcodev) [reply openapi/components/schemas/GatewayAccountConfig/Stripe.yaml:31]: It also will break Recomm I guess, I've checked a dozen of other definitions, we seems do not support objects for credentials, only scalars (string, boolean)

**Review by ashkarpetin — APPROVED** [review]

**Review by vova07 — APPROVED** [review]

**Review by iamdevice — APPROVED** [review]

**Pull request merged**